pandas/io/json/_json.py

```diff
@@ -19,8 +19,8 @@
 from pandas.io.formats.printing import pprint_thing
 from pandas.io.parsers import _validate_integer
 
-from .normalize import _convert_to_line_delimits
-from .table_schema import build_table_schema, parse_table_schema
+from ._normalize import _convert_to_line_delimits
```

Comment:
Not sure about renaming these modules as part of this (though maybe OK) - was this causing issues or are you just trying to make them more explicitly private?

Reply:
Trying to make them more explicitly private and match convention in io/excel and io/msgpack. Since we are not calling del any more in the __init__ file I thought this made sense

Reply:
Sure - might be fine just thinking through impacts but leave as is for now. I think if you define `__all__` in the init should take care of mypy errors

Reply:
do we actually need to rename modules for this to pass?

Reply:
Previously in the  __init__ file we were deleting the module reference - which the analyser didn’t like. We instead renamed the modules with a _ to make it obvious they are private. ( this seemed to match the convention in the io/excel package)

Reply:
ok, can you de-privatize convert_to_line_delimits then (as now the module itself is private)